test/unit/broadcast_active_set.c

```diff
@@ -56,40 +50,37 @@ int main(void)
         dst[i] = -1;
     }
 
-    for (i = 0; i < SHMEM_BCAST_SYNC_SIZE; i++)
-        bcast_psync[i] = SHMEM_SYNC_VALUE;
-
-    for (i = 0; i < SHMEM_BARRIER_SYNC_SIZE; i++) {
-        barrier_psync0[i] = SHMEM_SYNC_VALUE;
-        barrier_psync1[i] = SHMEM_SYNC_VALUE;
-    }
-
```

Comment:
I didn't move this block up with the declaration of barrier_pysnc0/1 because those variables are declared in the global space and this block is executed in main. However, I did wrap this block with #ifdef ENABLE_DEPRECATED_TESTS.